<testsuites name="results">
  <testsuite name="all" package="all">
    <property name="random_seed" value="1750968610" />
    <testcase name="test_counter_basic" classname="test_counter" file="/home/omare/Documents/UWASIC/Template/digital/test/test_counter.py" lineno="5" time="0.00031280517578125" sim_time_ns="0.001" ratio_time="3.196878048780488">
      <failure message="Test failed with RANDOM_SEED=1750968610" />
    </testcase>
    <testcase name="test_counter_reset" classname="test_counter" file="/home/omare/Documents/UWASIC/Template/digital/test/test_counter.py" lineno="31" time="7.295608520507812e-05" sim_time_ns="0.001" ratio_time="13.706875816993465">
      <failure message="Test failed with RANDOM_SEED=1750968610" />
    </testcase>
  </testsuite>
</testsuites>
